src/govuk/helpers/_links.scss

```
@@ -273,7 +273,7 @@
   // Force a colour change on hover to work around a bug in Safari
   // https://bugs.webkit.org/show_bug.cgi?id=224483
   &:hover {
-    color: rgba($govuk-text-colour, .99);
+    color: if($govuk-text-colour == inherit, inherit, rgba($govuk-text-colour, .99));
```

Given that the `:hover` state is effectively redundant when `$govuk-text-colour` is `inherit`, I wonder if we should just omit it entirely?

We can also check the type of `$govuk-text-colour` rather than testing for `inherit` specifically. Although I can't think of any other value that we would expect users to set it to, it's a better guard against `rgba()` being called with something that is not a colour.

Thoughts?

```suggestion
    @if (type-of($govuk-text-colour) == color) {
      color: rgba($govuk-text-colour, .99);
    }
```

---

Hi @36degrees.

I think you are right about testing for the type. I did think about whether there were other values we might want to guard against but wasn't sure how to achieve this.

I wasn't sure whether it was safe to omit the state entirely. You're knowledge of the CSS precedence is probably better than mine, so I'll defer to you on this point. (If it can be safely removed then, let's remove it.)

Thanks for the suggestion.